src/info_messages_linux.c

```diff
@@ -52,12 +53,16 @@ void get_info_messages(struct list_head *devices, unsigned *num_messages, const
 
   *messages = message_array;
   bool hasIntel = false;
+  bool hasMSM = false;
   bool hasAMD = false;
   struct gpu_info *gpuinfo;
   list_for_each_entry(gpuinfo, devices, list) {
     if (strcmp(gpuinfo->vendor->name, "Intel") == 0) {
       hasIntel = true;
     }
+    if (strcmp(gpuinfo->vendor->name, "msm") == 0) {
```

Comment:
If I understood correctly, fdinfo support for this driver was added in Linux version 6. Hence, it may be worth warning about this limitation.

Reply:
correct, v6.0 for GPU utilization (and v6.5 hopefully for memory usage)